CHANGELOG.md

```diff
@@ -36,6 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Breaking: `ReadSeekSource::new()` now takes `Settings`.
 - Breaking: Sources now use `f32` samples. To convert to and from other types of samples use
             functions from `dasp_sample` crate. For example `DaspSample::from_sample(sample)`.
+- Breaking: `symphonia::SeekError` has a new variant `AccurateSeekNotSupported`
+  and variants `Retrying` and `Refining` have been removed.
```

Comment:
Suggest to add: "catching this error may allow a caller to retry in coarse seek mode".